snippets/javascript/addeventlistener.md

```diff
@@ -0,0 +1,36 @@
+# addEventListener snippet in js
+
+## description
+
+> the addEventListener 'listens' to specific actions by the user and triggers a
```

Comment:
Very good description and clear comments, @aviv82 !

Reply:
thank you!